src/guide/security/storing-cryptographic-keys.md

```diff
@@ -0,0 +1,137 @@
+
+A variety of password managers can be used for the temporary storage of the user passwords and `SSH` keys. For the sake of clarity, [KeePass](https://keepass.info/) is used as an example password manager, specifically its [KeePassXC](https://keepassxc.org/) port running on Linux-based operating systems.
+
+For instructions on how to set up KeePassXC see the [Configuring KeePassXC](#configuring-keepassxc) section below.
+
+![KeePassXC: `Main` screen UI](../../img/KeePassXC.png)
+
+KeePassXC offers enhanced security, flexibility, and control. It not only stores passwords but also the `SSH` keys. When used for key storage, this password manager provides the `ssh-agent` with the stored keys, which are then promptly removed from its memory once the KeePassXC window is closed.
+
+::: tip
+
+Theoretically, any of the KeePass ports [listed on the official website](https://keepass.info/download.html) can be utilized for the key storage purposes.
+We recommend any of the following: [KeePassX](https://www.keepassx.org/) or [KeePassXC](https://keepassxc.org/).
+
+:::
+
+#### Configuring KeePassXC
+
+To configure KeePassXC, perform the following steps:
+
+1. Launch KeePassXC, then go to **Tools** > **Settings**, or select the **Gear** button from the top UI panel.
+
+2. In the **Application Settings** tab that appears, select **SSH Agent** from the left menu, and then select the **Enable SSH Agent integration** checkbox.
+
+   ::: info Show reference screenshot
+
+   ![KeePassXC `SSH Agent` tab: Enabling SSH Agent](../../img/keepassxc_ssh_agent.png)
+
+   :::
+
+3. Create a new KeePassXC Database. For instructions, see [KeePassXC User Guide > Creating Your First Database](https://keepassxc.org/docs/KeePassXC_UserGuide#_creating_your_first_database).
+
+4. For every key that you would like to store in the KeePassXC Database you created, perform the following steps:
+
+   - Add a new entry in the database. For instructions, see [KeePassXC User Guide > Creating Your First Database](https://keepassxc.org/docs/KeePassXC_UserGuide#_creating_your_first_database).
+
+   - When adding a new entry, attach the file containing the key by doing the following: select **Advanced** from the left menu, then select **Add** in the **Attachments** section, choose the required file in the **Select files** window that appears.
+
+   - When adding a new entry, select **SSH Agent** from the left menu, then select the key file you added from the **Attachment** menu in the **Private key** section; then select the following checkboxes:
+
+      - **Add key to agent when database is opened/unlocked**
+
+      - **Remove key from agent when database is closed/locked**
+
+      - **Require user confirmation when this key is used**
+
+   - If necessary, make other changes to the entry.
+
+   - When ready, select **OK** to save the entry.
+
+   ::: details Show reference screenshots
```

Comment:
```suggestion
   ::: details Here are reference screenshots
```

Reply:
This is an expandable section, "show" is a better option.